packages/next/build/webpack/plugins/flight-manifest-plugin.ts

```diff
@@ -187,29 +186,35 @@ export class FlightManifestPlugin {
           .filter((name) => name !== null)
 
         // Get all CSS files imported from the module's dependencies.
-        const visited = new Set()
+        const visitedCss = new Set()
```

Comment:
It's more accurate to be `visited`, as it keeps all visited modules not only CSS requests.